platform-enterprise_versioned_docs/version-25.1/enterprise/studios.mdx

```diff
@@ -52,6 +52,18 @@ Security scans and container inspection reports (including container specificati
 
 This guide assumes that all services will be run in the same container as the rest of your Seqera Platform services.
 
+If you were using Studios prior to GA (v25.1) please review the `tower.env` file and add the `TOWER_DATA_STUDIO_TEMPLATES_<TEMPLATE_NAME>_TOOL` variable. The configuration should be as follows: 
+
+```
+TOWER_DATA_STUDIO_TEMPLATES_<TEMPLATE_NAME>_STATUS: 'recommended'
+TOWER_DATA_STUDIO_TEMPLATES_<TEMPLATE_NAME>_REPOSITORY: 'public.cr.seqera.io/platform/data-studio-<template>:4.2.5-0.8'
+TOWER_DATA_STUDIO_TEMPLATES_<TEMPLATE_NAME>_ICON: '<template_name>'
+TOWER_DATA_STUDIO_TEMPLATES_<TEMPLATE_NAME>_TOOL: '<template_name>' 
```

Comment:
What is `<template_name>` in this example? Is it different from `<TEMPLATE_NAME>`?